### PR TITLE
[www] Adds cypress e2e testing 

### PR DESCRIPTION
**Pull request by shobhitchittora**
Adds Cypress test for www project.

#### Checklist

- [x] Cover Home Page navs
- [ ] Tests 404 links
- [ ] Add test for offline ( gatsby-offline-plugin )

Refs: #7421

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Hey! This is great! Instead of doing this on www (which takes a while to build), could you do it instead on a very simple site in the `integrations` folder?

Also it'd need an npm script that we run as part of travisci similar to https://github.com/gatsbyjs/gatsby/blob/f0ba07b9017dd117ec52828a47c96ccacbb0ebfc/.travis.yml#L29

File: www/cypress.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:8000",
+  "videoRecording": false
```
Comment (LeKoArts) [inline www/cypress.json]: According to the docs this is called `video`
https://docs.cypress.io/guides/references/configuration.html#Videos

Comment (shobhitchittora) [reply www/cypress.json]: @LeKoArts Nice catch! Fixed. 👍

**Comment by m-allanson** [comment]
Thanks @shobhitchittora! Should this be closed now that #7784 has been merged?

**Comment by m-allanson** [comment]
Great, thanks again!

**Pull request closed without merging**